shared/src/models/utils/rescue-resource.ts

```diff
@@ -32,13 +30,13 @@ export class PersonnelResource {
     @IsValue('personnelResource' as const)
     public readonly type = 'personnelResource';
 
-    @IsResourceDescription(personnelTypeAllowedValues)
-    public readonly personnelCounts: ResourceDescription<PersonnelType>;
+    @IsResourceDescription()
+    public readonly personnelCounts: ResourceDescription;
 
     /**
      * @deprecated Use {@link create} instead
      */
-    constructor(personnelCounts?: ResourceDescription<PersonnelType>) {
+    constructor(personnelCounts?: ResourceDescription) {
         this.personnelCounts = personnelCounts ?? {
             gf: 0,
             notarzt: 0,
```

Comment:
With dropping the fixed list of personnel types, I would prefer to also drop this default assignment.

I hope that nothing breaks by doing this, but I think TypeScript should catch every place where we access a specific personnelType without checking whether it is defined now that this ResourceDescription is string-based